src/sentry/issues/occurrence_consumer.py

```diff
@@ -289,6 +313,21 @@ def _process_message(
 
 
 class OccurrenceStrategyFactory(ProcessingStrategyFactory[KafkaPayload]):
+    def __init__(
+        self,
+        max_batch_size: int,
+        max_batch_time: int,
+        processes: int,
+        input_block_size: int,
+        output_block_size: int,
+    ):
+        super().__init__()
```

Comment:
Don't think it's needed
```suggestion
```

Reply:
I might just leave it in place. Doesn't hurt, and if we ever to add an init to the superclass it avoids problems